src/LGraphCanvas.ts

```diff
@@ -4011,10 +4011,8 @@ export class LGraphCanvas {
         return was_clicked
     }
     isAreaClicked(x: number, y: number, w: number, h: number, hold_click: boolean): boolean {
-        let pos = this.mouse
-        const hover = LiteGraph.isInsideRectangle(pos[0], pos[1], x, y, w, h)
-        pos = this.last_click_position
-        const clicked = pos && hover
+        const clickPos = this.last_click_position
+        const clicked = clickPos && LiteGraph.isInsideRectangle(clickPos[0], clickPos[1], x, y, w, h)
```

Comment:
Previously the pos used in inside check was `this.last_click_position`. https://github.com/Comfy-Org/litegraph.js/commit/20948f28cce33fd3491dea7d9d819789aa61a64e changes it to use `this.mouse`.

This commit changes it back to original logic.